### ISSUE TITLE
Suggestion: Improved Steam DB Expiry Handling

### ISSUE THREAD

**Issue by hk21702** [RimSort/RimSort#515]
Improve the handling of the steam database expiry to make it less confusing and alarming for typical users. Currently, handling of this warning is rather poor for the user experience. Current instructions to make it go away is to just go and increase the expiry time. This is suboptimal in that it is essentially circumventing what is supposed to be a feature in a rather round about and unintuitive manner at that. 

For instance, by default, if the user is GitHub hosted databases, don't flash a big warning that the DB is outdated unless their copy of the DB is outdated versus the remote DB.

Add a prompt in the warning dialogue to ignore this warning so that the user isn't spammed with it and they don't know how to ignore it.

**Comment by cebarks** [comment]
Default is now to use URL-based updates which should be a significantly more seamless experience for most users.
